CHANGELOG.md

```diff
@@ -1,5 +1,93 @@
 # [Versions](https://mui.com/versions/)
 
+## 5.14.12
+
+<!-- generated comparing v5.14.11..master -->
+
+_Oct 3, 2023_
+
+A big thanks to the 17 contributors who made this release possible. Here are some highlights ✨:
+
+- 🚀 Introduced color inversion utilities to Joy UI (#38916) @siriwatknp
+- 🚀 Added Chip and related TextField components to Material You @DiegoAndai, @mj12albert
+- 🔋 Added keyframes and TypeScript support to the zero-runtime packages @brijeshb42
```

Comment:
We still haven't released this package, I would not highlight changes related to it.

Reply:
I was even thinking that we probably shouldn't include changes about it in the changelog, but maybe it's fine in case people are interested.